File: LeetCode/Easy/0119-pascals-triangle-ii/0119-pascals-triangle-ii.go
func getRow(rowIndex int) []int {
    ret := make([]int, 1+rowIndex)
    ret[0] = 1
    
    for i := 1; i < rowIndex + 1; i++{
        temp := make([]int, i)
        copy(temp, ret)
        for j := 1; j < i + 1; j++ {
            if i == j { ret[j] = 1 }
            if i != j { ret[j] += temp[j-1] }
        }
    }
    return ret
}

File: LeetCode/Easy/0119-pascals-triangle-ii/0119-pascals-triangle-ii.py
class Solution:
    def getRow(self, rowIndex: int) -> List[int]:
        ret = [1]
        for i in range(1, rowIndex+1):
            temp = copy.deepcopy(ret)
            for j in range(1, i+1):
                if i == j:
                    ret.append(1)
                else:
                    ret[j] += temp[j-1]
        return ret